speech/snippets/multichannel.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command caption sends multichannel audio data to the Google Speech API
+// and prints its transcript.
+// go run multichannel.go ../testdata/commercial_stereo.wav
+package main
```

Comment:
Should be `package snippets` to match the other files in this directory. Also, we generally don't make binaries for snippets, instead relying on tests to ensure it works. Would you mind adding tests?

Reply:
changed package and added test.